hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/freon/TestOzoneClientKeyGenerator.java

```diff
@@ -16,21 +16,20 @@
  */
 package org.apache.hadoop.ozone.freon;
 
-import java.io.File;
-import java.io.FileOutputStream;
-import java.io.IOException;
-
+import org.apache.commons.io.FileUtils;
```

Comment:
Redundant change to this file.

Reply:
Reverting this file to a previous version.